.circleci/config.yml

```diff
@@ -185,6 +185,23 @@ jobs:
               --justfile ../../../single.just \
               simulate
 
+  just_simulate_eth-013-fp-upgrade-fjord:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: just simulate eth/013-fp-upgrade-fjord
+          command: |
+            go install github.com/mikefarah/yq/v4@latest
+            just install
+            cd tasks/eth/013-fp-upgrade-fjord
+            export SIMULATE_WITHOUT_LEDGER=1
+            just \
+              --dotenv-path .env \
+              --justfile ../../../nested.just \
+              simulate
```

Comment:
Two changes needed here:
1. This is a nested task, so we need to simulate as both the council and foundation
2. This currently doesn't run since it's not listed in this section of the CI file:

```yml
check_just_simulate:
    jobs:
      # Please add an invocation to `just simulate` if a ceremony is
      # active (e.g. it is the next ceremony to perform or you
      # expect the ceremony to work continuously), and remove it once
      # the ceremony is for historical archive only (e.g. the
      # ceremony is done).
      - just_simulate_sc_rehearsal_1
      - just_simulate_sc_rehearsal_2
      - just_simulate_sc_rehearsal_4
      - just_simulate_sep-base-001-MCP-L1
```

```suggestion
  just_simulate_eth-013-fp-upgrade-fjord:
    docker:
      - image: <<pipeline.parameters.ci_builder_image>>
    steps:
      - checkout
      - run:
          name: just simulate eth/013-fp-upgrade-fjord
          command: |
            go install github.com/mikefarah/yq/v4@latest
            just install
            cd tasks/eth/013-fp-upgrade-fjord
            export SIMULATE_WITHOUT_LEDGER=1
            just \
              --dotenv-path .env \
              --justfile ../../../nested.just \
              simulate council
            just \
              --dotenv-path .env \
              --justfile ../../../nested.just \
              simulate foundation
```

Reply:
Just unresolving to track item 1 in the comment above since that was not part of my suggest 😅

Reply:
Good to go now. Thanks.